app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -163,13 +162,7 @@ def apply_common_props(button, input)
 
   # Build single button
   def build_normal_button(bgi, index)
-    button_hide = hide_button?(bgi[:id])
-    if button_hide
-      # These buttons need to be present even if hidden as we show/hide them dynamically
-      return nil unless %w(timeline_txt timeline_csv timeline_pdf).include?(bgi[:id])
```

Comment:
_Just for the note:_

I don't quite understand, how is it possible, that the buttons were not hidden in the PR https://github.com/ManageIQ/manageiq/pull/9249, since there was still the condition that [`@report` can't be `nil`](https://github.com/ManageIQ/manageiq-ui-classic/blame/9bf3726208449444b5abb904a1486ecf3194f043/app/helpers/application_helper/toolbar_builder.rb#L860-L865) to display the buttons.

On the same page today, with the same condition in [`timeline_download` button](https://github.com/ManageIQ/manageiq-ui-classic/commit/93fbc678025c7bbcd1fbfb51552bfb422fdd9b1d#diff-979763b473d1dc9d593f4c8b9f68ef8fR8), the toolbar buttons are hidden for me.
![screenshot from 2017-03-02 23-03-14](https://cloud.githubusercontent.com/assets/1187051/23529146/be0148be-ff9c-11e6-8eb3-4f7847cbb72a.png)

But when I've removed the condition, the buttons were displayed correctly, so I guess there is no issue now.